docs/usage/getting_started.md

```diff
@@ -0,0 +1,28 @@
+### Get Started
+
+There are many ways to get started with flagd. Choose the method that best serves your requirements to get started.
+
+### Docker
+
+1. `docker pull ghcr.io/open-feature/flagd:latest`
+
+### Go binary
+
+1. Install Go 1.18 or above
+1. run `go install github.com/open-feature/flagd@latest`
+
+### Release binary
+
+1. Download pre-built binaries from https://github.com/open-feature/flagd/releases
+
+### Snapcraft
+
+Install via Snapcraft, documentation can be found here: https://snapcraft.io/flagd
+
+### Systemd service
+
+Documentation for installing flagd as a systemd service can be found here: https://github.com/open-feature/flagd/blob/main/docs/other_resources/systemd_service.md
+
+### Open Feature Operator
+The OpenFeature Operator is a Kubernetes native operator that allows you to expose feature flags to your applications. It injects a flagD sidecar into your pod and allows you to poll the flagD server for feature flags in a variety of ways.
+To get started with the operator, view the project here: https://github.com/open-feature/open-feature-operator
```

Comment:
What is your opinion on introducing a footer for linking the adjacent pages in the table of contents? It may help make the docs pages feel more rounded